tools/java/org/unicode/cldr/util/CLDRPaths.java

```diff
@@ -11,6 +11,7 @@
  * These must not be used by any code destined for the SurveyTool, as this class will not be included.
  * @author srl
  *
+ * TODO: clarify "this class will not be included" comment above -- no longer true?
```

Comment:
I think that ST has to get paths via CldrConfig.java. Steven can confirm.

Reply:
Other than that, the rest looks good to me.

Reply:
Using CLDRPaths.BASE_DIRECTORY does seem to work OK. I've just tried replacing it with CLDRConfig.getInstance().getCldrBaseDirectory() and that also seems to work OK. So, I'll make that change.

The comment "this class will not be included" could still use some clarification. Is avoiding CLDRPaths in ST necessary, or just preferable. If necessary, why necessary? If preferable, why preferable? In the meantime I'll delete "-- no longer true?" from my comment.

Reply:
Avoiding CLDRPaths is necessary, not preferable.  Because CLDRPaths are staticly configured paths which assume a single instance. Not true for SurveyТооl

Reply:
I'm not sure what you mean by "single instance" -- instance of what? Are there ever multiple instances of SurveyТооl running on the same server with different values for BASE_DIRECTORY?

Reply:
SurveyTool has to read multiple base directories, yes (multiple versions). CLDRPaths seems convenient but is not flexible enough.